arithmetization/src/test/java/net/consensys/linea/zktracer/ReplayTests.java

```diff
@@ -199,84 +203,346 @@ void replayMultipleReplayFiles() {
     replayBulk("");
   }
 
+  // @Disabled
   @Test
   void leoFailingRange() {
-    replay("5389571-5389577.json.gz");
+    replay("5389571-5389577.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void failingMmuModexp() {
-    replay("5995162.json.gz");
+    replay("5995162.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void failRlpAddress() {
-    replay("5995097.json.gz");
+    replay("5995097.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void rlprcptManyTopicsWoLogData() {
     replay("6569423.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void multipleFailingCallToEcrecover() {
-    replay("5000544.json.gz");
+    replay("5000544.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void incident777zkGethMainnet() {
-    replay("7461019-7461030.json.gz");
+    replay("7461019-7461030.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void issue1006() {
-    replay("6032696-6032699.json.gz");
+    replay("6032696-6032699.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void issue1004() {
     replay("6020023-6020029.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void block_6110045() {
     // The purpose of this test is to check the mechanism for spotting divergence between the replay
     // tests and mainnet.  Specifically, this replay has transaction result information embedded
     // within it.
-    replay("6110045.json.gz");
+    replay("6110045.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void failingCreate2() {
-    replay("2250197-2250197.json.gz");
+    replay("2250197-2250197.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void blockHash1() {
-    replay("8718090.json.gz");
+    replay("8718090.json.gz", false);
   }
 
+  // @Disabled
```

Comment:
why all teg @Disabled are commented ? Better delete them no ?

Reply:
It's just easier to uncomment them again when they are already in the file